rootdir/system/etc/media_profiles.xml

```diff
@@ -38,7 +38,7 @@
 <!ATTLIST Audio sampleRate CDATA #REQUIRED>
 <!ATTLIST Audio channels (1|2) #REQUIRED>
 <!ELEMENT ImageEncoding EMPTY>
-<!ATTLIST ImageEncoding quality (90|80|70|60|50|40) #REQUIRED>
+<!ATTLIST ImageEncoding quality (100|100|100|100|100|100) #REQUIRED>
```

Comment:
Yes. camera uses different jpeg quality for every condition. This forces camera to always take photos at 100% jpeg quality.(Although maximum jpeg quality on camera hal is 85).